src/vsgSDL/window.cpp

```diff
@@ -97,10 +99,10 @@ namespace vsgSDL {
     }
 
     bool Window::initSDLVulkan(std::string dynLib) const {
-        if(SDL_Vulkan_LoadLibrary(dynLib.c_str()) == -1)
-            return false;
-
-        return true;
+        if (dynLib == "")
+            return SDL_Vulkan_LoadLibrary(NULL) != -1;
+        else 
+            return SDL_Vulkan_LoadLibrary(dynLib.c_str()) == -1;
```

Comment:
I find a mistake here, it should be `SDL_Vulkan_LoadLibrary(dynLib.c_str()) != -1`